src/rat/rat_basin.py

```diff
@@ -597,9 +597,10 @@ def rat_basin(config, rat_logger):
                 else: 
                     raise Exception('Step-9 was not run OR There was an error in creating reservoir shapefile using spatial join for this basin from the global reservoir vector file.')
             # Get Sarea
+            filt_options = config['GEE']['bot_filter']
             run_sarea(gee_start_date.strftime("%Y-%m-%d"), config['BASIN']['end'].strftime("%Y-%m-%d"), sarea_savepath, 
-                                                                                    basin_reservoir_shpfile_path, reservoirs_gdf_column_dict)
-            GEE_STATUS = 1
+                                                                                    basin_reservoir_shpfile_path, reservoirs_gdf_column_dict,filt_options)
```

Comment:
Make sure the function run_sarea works with filt_options as None.

Reply:
Fixed. Also added None as default argument value in run_sarea